sources/org.osbuild.ostree

```diff
@@ -81,17 +80,6 @@ SCHEMA = """
 """
 
 
-def ostree(*args, _input=None, **kwargs):
-    args = list(args) + [f'--{k}={v}' for k, v in kwargs.items()]
-    print("ostree " + " ".join(args), file=sys.stderr)
-    subprocess.run(["ostree"] + args,
-                   encoding="utf-8",
-                   stdout=subprocess.PIPE,
-                   stderr=subprocess.STDOUT,
```

Comment:
Not sure it matters, but this is different from what ostree.cli() does, as it prints stdout to stderr, whereas this captures stdout. So there will be more output noise with the new version.

Reply:
yeah. LMK if that is a deal breaker.

Reply:
I have no idea how the stability of log output is to the project, someone else will have to answer that. Just noticed the fact that it changed.

Reply:
I think it doesn't make much of a difference.  When storing the output to return to a caller (like in the case of `--json`) we store both the stdout and stderr from stages and return it as a log.  Otherwise, we just print it directly, after running it through a `monitor` class.

There shouldn't be an issue with changing a stage's stdout/stderr output, but it's a good idea to make sure it's printed, one way or another, for troubleshooting.